src/createServer.js

```diff
@@ -1,3 +1,59 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+const {
+  NO_TEXT_ERROR_MESSAGE,
+  NO_CASE_ERROR_MESSAGE,
+  STATIC_CASES,
+  INVALID_CASE_ERROR_MESSAGE,
+} = require('./constatnts');
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    const url = new URL(req.url, 'http://localhost:5700');
```

Comment:
What would you do if the port is changed? It is a hardcoded value and needs to be fixed. You can get a port from res, here is an example    ` const serverLink = `http://${req.headers.host}`;`